src/Tests/API/APIApprovals.ApproveAzureServiceBusTransport.approved.txt

```diff
@@ -48,6 +48,7 @@
     public interface ISanitizationStrategy
     {
         string Sanitize(string entityPathOrName, NServiceBus.AzureServiceBus.EntityType entityType);
+        void SetDefaultRules(NServiceBus.Settings.SettingsHolder settings);
```

Comment:
I don't like this "new design". It's not a _sanitization strategy_ concern setting up default values. What a _sanitization strategy_ has to do is to sanitize.
Moreover, if a user wants to implement a new strategy, they need to know what to do to setup default rules and then go through the transport code to learn which `setting keys` have to configure